src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -131,6 +131,16 @@ public static String sanitizeForHtml(String str) {
     }
 
     /**
+     * Escapes HTML tag safely. This function can be applied multiple times.
+     */
+    public static String sanitizeForHtmlTag(String str) {
+        if (str == null) {
+            return null;
+        }
+        return str.replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;");
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8078323/14144021/b03a6de0-f6c0-11e5-9536-3f3f5ce3f0c7.png)

@wkurniawan07 This is what I get. I'm not sure why they are different. But the issue here is if we don't handle `&`, it may create extra opening or closing tags that can break the HTML structure on the next time we read HTML output to compare with the page source.